eth/__init__.py

```diff
@@ -2,14 +2,14 @@
 import sys
 
 from eth.tools.logging import (
-    setup_trace_logging
+    setup_ethereum_logging
 )
 
 #
 #  Setup TRACE level logging.
 #
 # This needs to be done before the other imports
-setup_trace_logging()
+setup_ethereum_logging()
```

Comment:
Along with the rename, maybe this should just be `setup_logging` or `setup_extended_logging`